build_support/build_deps.sh

```diff
@@ -8,14 +8,20 @@ if [ -e /.dockerenv ];then
     sudo yum swap -y -q -- remove fakesystemd -- install systemd systemd-libs
 fi
 
-sudo yum install -q -y wget tar gcc gcc-c++ make git patch openssl-devel.x86_64 flex flex-devel.x86_64 byacc xz-devel libgcrypt-devel.x86_64  libidn-devel.x86_64 ncurses-devel.x86_64 autoconf automake libtool java-1.8.0-openjdk-devel.x86_64 
+sudo yum install -q -y wget tar gcc gcc-c++ make git patch openssl-devel.x86_64 flex flex-devel.x86_64 byacc xz-devel libgcrypt-devel.x86_64  libidn-devel.x86_64 ncurses-devel.x86_64 autoconf automake libtool java-1.8.0-openjdk-devel.x86_64
 
 wget https://cmake.org/files/v3.10/cmake-3.10.0-Linux-x86_64.tar.gz
 tar zxf cmake-3.10.0-Linux-x86_64.tar.gz
 
 echo 'export PATH='$PWD'/cmake-3.10.0-Linux-x86_64/bin/:$PATH' >> profiles/cmake.profile
 source ./profiles/cmake.profile
 
+wget https://www.samba.org/ftp/ccache/ccache-3.3.4.tar.gz -O ccache-3.3.4.tar.gz
+tar zxvf ccache-3.3.4.tar.gz
+pushd ccache-3.3.4
+./configure && make && make install
+popd
```

Comment:
Isn't ccache already in CentOS's yum repo?

Reply:
yes

Reply:
If it is in the yum repo, why don't you just install it by using yum?

Reply:
no, I say yum repo can not find ccache.